### ISSUE TITLE
App not responding on tapping Manage Conference button.

### ISSUE THREAD

**Issue by infinity-plus** [Chooloo/koler#425]
## Checklist

- [x] I am using the latest version - v1.4.5 (F-Droid Build) 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1. Make or receive a call via Koler.
2. Merge another call to make it a conference call
3. Tap on Manage Conference button.

### Actual behavior

When you tap on manage conference button to see who all are there in a conference call, it does nothing, and app stops responding after a while. You cannot even disconnect the call. I had to ask the other person to do it.

### Expected behavior

Upon tapping Manage Conference button, a list of numbers/contacts in the conference shows up, with a disconnect button next to each row.

### Device info
 - Android version: 11
 - Device model: Motorola moto G40 Fusion

**Comment by mikenrafter** [comment]
I've experienced this bug too, though the conference did actually merge, and I was able to hear all participants.

**Comment by infinity-plus** [comment]
> I was able to hear all participants.
Of course, I'm talking about managing a conference. One should be able to disconnect people in conference if they're the host. The app just freezes leaving you on a conference call. No mute, no disconnect works.
